setup.cfg

```diff
@@ -21,7 +21,7 @@ classifiers =
 [options]
 include_package_data = True
 install_requires =
-    Django<4.2
+    Django==4.2
```

Comment:
We usually don't pin dependencies for libraries:

```suggestion
    Django>=4.2
```

Reply:
Afterthought: as we only change things in the sandbox, maybe we can remove constraints on the `Django` dependency for the `django-lti-toolbox` package and pin Django release in the sandbox. WDYT?

Reply:
Following our discussion @jmaupetit, I have fixed up my commit, WDYT? I won't forget to update the initial commit body while squashing my commits.